docs/integration-guides/key-management.md

```diff
@@ -227,11 +227,11 @@ curl -d '{
     When not following this guide closely, the following **inappropriate sequence of events could lead to erroneous amounts sent** to a recipient.
 
     1. An account's balance, say 5 $nano$, was obtained using the [`account_balance`](/commands/rpc-protocol#account_balance) RPC command (**never use this command for transaction related operations**). This balance is valid as of hypothetical **BLOCK_A**.
-    1. By another process you control, a receive (**BLOCK_B**) was signed and broadcasted into your account-chain (race-condition).
+    2. By another process you control, a receive (**BLOCK_B**) was signed and broadcasted into your account-chain (race-condition).
```

Comment:
With the markdown, you don't need to explicitly number the items, just putting a 1. in front of all of them indicates it will be an ordered list. Then later if you need to add something in the middle, you don't have to rearrange the previously set numbers, just add a new one in. We should adjust these back for that reason.

Reply:
Reverted